django_celery_beat/migrations/0010_auto_20190428_1204.py

```diff
@@ -0,0 +1,43 @@
+# Generated by Django 2.0.5 on 2019-04-28 12:04
```

Comment:
just to avoid risk, would you mind removing this migration, pull from master, and generate new migrations with django 2.2 for this pr and push again to make Travis green again before we merge?

Reply:
ok

Reply:
done this